panda_simulator:
  panda_gripper_controller:
    type: panda_sim_controllers/PandaGripperController
    joint_state_topic:  /franka_gripper/joint_states
    homing_topic: /franka_gripper/homing
    stop_topic: /franka_gripper/stop
    grasp_topic: /franka_gripper/grasp
    move_topic: /franka_gripper/move
    joints:
          # main joint
          panda_finger_joint1_controller:
            joint: panda_finger_joint1
            pid: {p: 5000,  i: 50, d: 0.5}
          # mimic joint
          panda_finger_joint2_controller:
            joint: panda_finger_joint2
            pid: {p: 5000,  i: 50, d: 0.5}
